app/controllers/navigation/Navigator.scala

```diff
@@ -83,7 +84,8 @@ object Navigator {
   val standard: PartialFunction[DeclarationPage, Mode => Call] = {
     case DeclarantDetails            => routes.AdditionalDeclarationTypeController.displayPage
     case ConsignmentReferences       => routes.DeclarantDetailsController.displayPage
-    case DeclarantIsExporter         => routes.ConsignmentReferencesController.displayPage
+    case LinkDucrToMucr              => routes.ConsignmentReferencesController.displayPage
+    case Mucr                        => routes.LinkDucrToMucrController.displayPage
```

Comment:
Could all these be moved to `common` partial function?

Reply:
Good spot. Confused :confused:

Reply:
May need to rework Navigator class to be more readable. It was fine at the beginning when journeys were mostly the same, because the specific parts were rather small. Now it's the opposite :/